cryptoapis/README.md

```diff
@@ -67,36 +87,6 @@ The adapter takes the following environment variables:
 }
 ```
 
-### Height
```

Comment:
Left this as it is @justinkaseman, because probably it could be batch-updated along other balance endpoint docs.